### PR TITLE
Don't require super user

### PR DESCRIPTION
**Pull request by ernestii**
* Don't require a superuser to run
* Remove unused libraries
* Remove `bluebird`

I tested main functionality and it works. You might need to remove `~/.logdna.conf` (to recreate is under regular user, otherwise you'll get a `eacces` error).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -37,425 +34,398 @@ program
...
-        } else {
-            resolve(false);
-        }
+properties.parse(config.DEFAULT_CONF_FILE, { path: true }, (err, parsedConfig) => {
+    if (err) {
+        return utils.log('Unable to read the configuration file.');
+    }
+    config = Object.assign(config, parsedConfig || {});
+
+    utils.performUpgrade(config, error => {
```
Comment (dchai76) [inline index.js]: Can you parenthesize `error`? https://logdna.slite.com/app/channels/BJNKjTxAG/notes/e7mpztz~Tu#5e369002

File: index.js [hunk 2]
```diff
@@ -37,425 +34,398 @@ program
...
+        if (error) utils.log(error);
```
Comment (dchai76) [inline index.js]: I personally prefer always bracketing `if` statements, less error-prone when refactoring.

File: index.js [hunk 3]
```diff
@@ -37,425 +34,398 @@ program
...
-checkElevated()
-    .then(isElevated => {
-        if (!isElevated) {
-            console.log('You must be an Administrator (root, sudo) run this agent! See -h or --help for more info.');
-            process.exit();
-        }
+    program.command('register [email]')
+        .description('Register a new LogDNA account')
+        .action(function(email) {
+            var nextstep = function(email) {
```
Comment (dchai76) [inline index.js]: Can we `const` this? Also make it an arrow function? This goes for the all the functions below; I know you're just moving stuff around but might as well refactor as we go.

Comment (dchai76) [reply index.js]: IMO this code would be more readable if we pulled all the action functions out and declared them elsewhere, but up to you.

File: index.js (hunk 1 above)
Comment (ernestii) [reply index.js]: I'm going to work on that stuff later :)

**Pull request merged**